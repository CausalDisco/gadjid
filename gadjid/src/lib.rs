// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
