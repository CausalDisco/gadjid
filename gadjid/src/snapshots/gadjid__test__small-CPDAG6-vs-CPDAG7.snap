---
source: src/lib.rs
expression: "test(&format!(\"200{:0>2}.CPDAG-10\", true_id),\n    &format!(\"200{:0>2}.CPDAG-10\", guess_id))"
---
g_true: 20006.CPDAG-10
g_guess: 20007.CPDAG-10
ancestor_aid:
  - 0.12222222222222222
  - 11
oset_aid:
  - 0.12222222222222222
  - 11
parent_aid:
  - 0.13333333333333333
  - 12
shd:
  - 0.13333333333333333
  - 6
T:
  - 2
  - 3
  - 4
  - 5
  - 7
NAM: []
Y: 1
random_adj:
  - 0
  - 6
  - 8
proper_ancestors:
  - 1
oset: []
possible_descendants:
  - 2
  - 3
  - 4
  - 5
  - 7
parent_adjustment_NVA:
  - 0
  - 8
  - 9
ancestor_adjustment_NVA:
  - 0
  - 8
  - 9
nondescendant_adjustment_NVA:
  - 0
  - 1
  - 6
  - 8
  - 9
oset_adjustment_NVA:
  - 0
  - 8
  - 9
empty_adjustment_NVA:
  - 0
  - 8
  - 9
random_Z_adjustment_NVA:
  - 0
  - 6
  - 8
  - 9
